instance/settings.py

```diff
@@ -34,6 +34,7 @@
 LOGFILE = 'error.log'
 #: File uploads
 UPLOADED_FILES_DEST = 'imgee/static/uploads'
+UPLOADED_FILES_DIR = 'uploads'  # this folder should be inside `imgee/static/`
 #: S3 Configuration
 AWS_ACCESS_KEY = 'Set aws access key here'
 AWS_SECRET_KEY = 'Set aws secret key here'
```

Comment:
We don't want to encourage inserting values into a file that's version controlled in git. Maybe this file should be named `settings-sample.py`.